managers/templates/managers/projects_list.html

```diff
@@ -28,6 +30,13 @@ <h1>
                         <img src="{% static 'managers/images/ico-07.png' %}">
                     </a>
                 {% endif %}
+
+                {% trans 'Task activities' %}
```

Comment:
I think that this should be removed. This cause that UI for project list looks odd.

Reply:
Changed